README.md

```diff
@@ -561,6 +561,58 @@ Use it to type in your key and install.
 
 Go back to Step 4A-4 and then finish with Step 4A-5.
 
+---
+## Forks
+
+### Which Fork Should I Use?
+
+> [!WARNING]
+> If you have to ask, you're not ready for forks.
+
+Begin your research in Discord #custom-forks
+
+Please do not ask about forks outside of that channel.
+
+Please start with `commaai/master-ci`. This is the latest upstream and supports lateral only. 
```

Comment:
```suggestion
If you're new, please start with `commaai/master-ci` and use it for two weeks. This is the latest official version with only lateral support. This will give you a good baseline to compare the other forks to.
```